dockerfiles/theia/Dockerfile

```diff
@@ -72,7 +72,7 @@ RUN che:theia init -c ${HOME}/che-theia-init-sources.yml
 RUN che:theia cdn --theia="${CDN_PREFIX}" --monaco="${MONACO_CDN_PREFIX}"
 
 # Compile Theia
-RUN yarn
+RUN yarn global add node-gyp && node-gyp install & yarn
```

Comment:
why adding node-gyp there ? it's not used

Reply:
```
RUN yarn global add node-gyp && node-gyp install & yarn
```

is same as

```
RUN yarn global add node-gyp && node-gyp install
RUN yarn
```

It's safer to call `node-gyp install` before all `yarn` calls.

Reply:
... and I found a typo there. I'll fix it and re-push.